tests/test_update_agent_key.py

```
@@ -0,0 +1,60 @@
+"""LiteLLM virtual-key re-scope (update_agent_key) — keystone for synced model management."""
+from __future__ import annotations
+
+import pytest
+from tinyagentos.llm_proxy import LLMProxy
+
+
+class _Resp:
+    def __init__(self, status): self.status_code = status; self.text = ""
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)